license-header.txt

```diff
@@ -0,0 +1,13 @@
+Copyright {{attrs.git_file_created_year}} {{props["copyrightOwner"]}}
```

Comment:
nit: Can we move the header to some specified directory? For example, `hack/`?

Reply:
fixed